README.md

```diff
@@ -1 +1,148 @@
-# ruby-TDD
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Ruby TDD <a name="about-project"></a>
+
+**Ruby TDD** is a basic project that uses Ruby language & Rspec library to demonstrate TDD techniques
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **Ruby**
+- **Rspec**
+
+### Key Features <a name="key-features"></a>
+
+- **Test Driven Development**
+- **Factorial method**
+- **FizzBuzz method**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Live Demo is not available yet
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Example command:
+
+```sh
+ gem install ruby
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  git clone git@github.com:hmunish/ruby-TDD.git
+  cd ruby-TDD
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Create objects of classes & run on code editor
+
+### Run tests
+
+To test the project, execute the following command:
+
```

Comment:
- [ ] [optional] @Team it would have been nice if you added the commands to run the project and tests in the project.